faq.rst

```diff
@@ -0,0 +1,90 @@
+Frequently Asked Questions
+==========================
+
+The following FAQs are organized into
+
+- a "Components" section that includes each of the seven PURL components
+  (``scheme``, ``type``, ``namespace``, ``name``, ``version``, ``qualifiers``
+  and ``subpath``), and
+
+- a "General" section containing a mix of questions and answers that don't fit
+  neatly into a component-focused category.
+
+If you have a question about the PURL specification and don't find an answer
+below, you can open an issue `here <https://github.com/package-url/purl-spec/issues/new?template=Blank+issue>`_.
+
+Components
```

Comment:
Please include only the relevant section, no placeholders.

Reply:
Placeholders removed.